practices/documentation_testing.md

```diff
@@ -0,0 +1,105 @@
+
+## Why You Might Want Documentation Testing?
+Through testing your documentation you understand the first impression of your product. Moreover, you gain insight into the decision funnel and you start to understand who is using your product and why. Documentation testing can help you find weak points in your documentation.
+
+It is a series of good practices and even small teams without tech writers can improve their developer experience by adopting documentation testing.
+
+## Problems Documentation Testing Helps to Solve
+
+* Lack of understanding around your product
+* Lack of developers using your product
+* Damaged information flow
+* Unclear instructions
+* Lack of empathy towards developers
+
+## How to Implement Documentation Testing?
+There are many ways of implementing Documentation Testing. You can start with: 
+### Implementing Passive Feedback Gathering
+Consider the following Passive Feedback Gathering methods:
+#### Ratings on Existing Docs
+Start by adding a way to rate your existing documentation. Consider adding questions like: “Is this page helpful to you?” Or “Did this guide benefit you?”. Even a simple star rating can add crucial data for your decisions.
+> Allowing ratings on your Docs helps to build rapport with your readers. Look at which pages are well-received and which need improvements.
+
+#### On-Page Behaviour
+The behavior on-page can tell you a lot about your Doc’s perception. Start using analytical software (popular options are: [Google Analytics](http://analytics.google.com), [GoSquared](https://www.gosquared.com/analytics/), [Clicky](https://clicky.com), [Statcounter](https://statcounter.com), or [Matomo](https://matomo.org)) to monitor your readers. 
+> The time spent on each page, the bounce rate of your pages, referrals, and leaving pages can help you understand how people interact with your docs.
+
+#### Off-page Behaviour
+Understanding who is your reader is crucial to tailor your docs around them. Software like [Google Search Console](https://search.google.com/search-console/about) can show you why are your readers visiting your documentation.
+> Tracking why people visit you is crucial to understand which issues and questions bring people to you. Knowing what brings people to you, you can tailor your docs to provide a better experience to your visitors.
+
+#### Clicks and download tracking
+Consider logging how many people download your code samples. For this logging, you can use software like [Google Tag Manager](http://tagmanager.google.com). With Google Tag Manager, you can use Custom events, for example scroll rate or download rate, to gain better insight on your Docs performance.
+> Track any action that is important for you. From Sing-ups to finishing demo, this information directly makes difference in improving developer experience.
+
+### Implementing Active Testing
+Following Active Testing methods can give you insight on your Documentation DX.
```

Comment:
Of course that Active Testing can give me insight on my documentation DX... but what does it mean "active testing" in general? Is that clear for everyone or we should write a sentence what difference is in active testing against passive?